tools/packaging/kata-deploy/kata-cleanup/base/kata-cleanup-stable.yaml

```diff
@@ -1,46 +0,0 @@
----
```

Comment:
I understand the version bump in the version file, but while are you dropping this file. Feel like this deserves its own separate commit with an explanation of why this file is being dropped.

Reply:
Wow, thanks for noticing it!. I didn't drop the file and I'm not sure why it was removed. The commit was auto-generated by `./update-repository-version.sh`. Let me see what happened.

Reply:
Archana, this is an automated part of the release that was introduced with https://github.com/kata-containers/kata-containers/pull/2453

In that PR we discussed a few things, one of those is how to treat `:latest`, `:stable`, and "specifc-versions" tags.  This file being dropped is the agreement of that discussion, as once we branch we want folks who are using that specific branch to have a stable link to kata-deploy, and with that avoid surprises when installing it.

Again, this is automated as part of the release scripts, and so do is the upload of the images using the correct tags.

Reply:
@bergwolf, that's actually expected. Here's the commit where this was introduced: https://github.com/kata-containers/kata-containers/pull/2453/commits/43a72d76e2520ae044335b9f1dbf54c12363f54b

The important part of that is this part of the commit message:
"""
Let's teach our `update-repository-version.sh` script to properly update
the kata-deploy tags on both kata-deploy and kata-cleanup yaml files.

The 3 scenarios that we're dealing with, based on which branch we're
targetting, are:

```
 1) [main] ------> [main]        NO-OP
   "alpha0"       "alpha1"

                   +----------------+----------------+
                   |      from      |       to       |
  -----------------+----------------+----------------+
  kata-deploy      | "latest"       | "latest"       |
  -----------------+----------------+----------------+
  kata-deploy-base | "stable        | "stable"       |
  -----------------+----------------+----------------+

 2) [main] ------> [stable] Update kata-deploy and
   "alpha2"         "rc0"   get rid of kata-deploy-base

                   +----------------+----------------+
                   |      from      |       to       |
  -----------------+----------------+----------------+
  kata-deploy      | "latest"       | "rc0"          |
  -----------------+----------------+----------------+
  kata-deploy-base | "stable"       | REMOVED        |
  -----------------+----------------+----------------+

 3) [stable] ------> [stable]    Update kata-deploy
    "x.y.z"         "x.y.(z+1)"

                   +----------------+----------------+
                   |      from      |       to       |
  -----------------+----------------+----------------+
  kata-deploy      | "x.y.z"        | "x.y.(z+1)"    |
  -----------------+----------------+----------------+
  kata-deploy-base | NON-EXISTENT   | NON-EXISTENT   |
  -----------------+----------------+----------------+
```
"""

Reply:
Tried again, and it's the same. Looks like the script does do it intentionally. See 43a72d76e2520ae044335b9f1dbf54c12363f54b